rust/automerge-c/src/doc/mark.rs

```diff
@@ -80,14 +108,14 @@ pub unsafe extern "C" fn AMmarkName(mark: *const AMmark) -> AMbyteSpan {
             return name;
         }
     }
-    AMbyteSpan::default()
+    Default::default()
 }
 
 /// \memberof AMmark
-/// \brief Gets the value of the mark
+/// \brief Gets the value of a mark.
 ///
 /// \param[in] mark A pointer to an `AMmark` struct.
-/// \return A pointer to an `AMresult` struct with an `AMitem` struct.
+/// \return A pointer to an `AMresult` struct with an `AM_VAL_TYPE_MARK` item.
```

Comment:
@jkankiewicz The value can be of any type (string/number/etc.) so I think the previous doc was better.

Reply:
> The value can be of any type (string/number/etc.) so I think the previous doc was better.

@ConradIrwin, you're right, it's a mistake to identify it as an `AM_VAL_TYPE_MARK` item.